src/server/repositories/UserRepository.ts

```diff
@@ -160,30 +141,28 @@ export class UserRepository implements UserRepositoryInterface {
         },
       ],
     }
-    const whereConditions: any =
+    let whereConditions: any =
       conditions.searchText.length > 0
         ? {
             ...searchTextCondition,
             userId: conditions.userId,
           }
         : { userId: conditions.userId }
+    if (conditions.tags && conditions.tags.length > 0) {
+      const searchTagConditions = {
+        [Op.or]: conditions.tags.map((tag) => {
+          return { tagStrings: { [Op.substring]: `${tag}` } }
+        }),
+      }
+      whereConditions = { ...whereConditions, searchTagConditions }
```

Comment:
was just testing it locally, searching with tags breaks without the addition of the spread operator `...` below:
```suggestion
      whereConditions = { ...whereConditions, ...searchTagConditions }
```
even with the above in place though, I'm worried that the `[Op.or]` from `searchTagConditions` will override `[Op.or]` from `searchTextCondition` defined above, if the user searches by both `searchText` and `tags`

Reply:
we actually don't support searchText and Tags at the same time.